esp/platform/espcontext.cpp

```diff
@@ -466,9 +466,6 @@ class CEspContext : public CInterface, implements IEspContext
     {
         if (m_txSummary)
         {
-            if (!m_hasException && (getEspLogLevel() <= LogNormal))
```

Comment:
I made this change based on the discussions among some of us. Without this change, TxSummary will be cleaned and we lost the data someone has already added into the TxSummary. @afishbeck any comment?